remoulade/api/main.py

```diff
@@ -0,0 +1,42 @@
+""" This file describe the API to get the state of messages """
+from flask import Flask, jsonify, make_response, request
+from werkzeug.exceptions import BadRequest, NotFound
+
+import remoulade
+from remoulade.state import StateNamesEnum
+
+app = Flask(__name__)
+
+
+@app.route("/messages/states/")
```

Comment:
generally there is no / at the end of routes, so `/messages/states` would be better